GeneratorConfig/Categories.json

```diff
@@ -13,6 +13,11 @@
         "Name": "PokemonBDSP",
         "Display": "BDSP",
         "Settings": [ "PokemonBDSP_Settings" ]
+    },

```

Comment:
I think you did not remove the files you wanted to remove (as this Pull Request seems to contains all your changes)

Reply:
Shoot, I’ll fix that

Reply:
I've removed all of the SV Native Programs